app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Here also, the association for the Like should be using a `author_id` column. Hence, Please apply `belongs_to :author, class_name: 'User'`